js/tests/index.html

```diff
@@ -2,138 +2,94 @@
     <script src="../../docs/assets/js/vendor/tether.min.js"></script>
-    <script>
-      // Disable jQuery event aliases to ensure we don't accidentally use any of them
-      (function () {
-        var eventAliases = [
-          'blur',
-          'focus',
-          'focusin',
-          'focusout',
-          'load',
-          'resize',
-          'scroll',
-          'unload',
-          'click',
-          'dblclick',
-          'mousedown',
-          'mouseup',
-          'mousemove',
-          'mouseover',
-          'mouseout',
-          'mouseenter',
-          'mouseleave',
-          'change',
-          'select',
-          'submit',
-          'keydown',
-          'keypress',
-          'keyup',
-          'error',
-          'contextmenu',
-          'hover',
-          'bind',
-          'unbind',
-          'delegate',
-          'undelegate'
-        ]
-        for (var i = 0; i < eventAliases.length; i++) {
-          var eventAlias = eventAliases[i]
-          $.fn[eventAlias] = function () {
-            throw new Error('Using the ".' + eventAlias + '()" method is not allowed, so that Bootstrap can be compatible with custom jQuery builds which exclude the "event aliases" module that defines said method. See https://github.com/twbs/bootstrap/blob/master/CONTRIBUTING.md#js')
-          }
-        }
-      })()
-    </script>
 
     <!-- QUnit -->
     <link rel="stylesheet" href="vendor/qunit.css" media="screen">
     <script src="vendor/qunit.js"></script>
+
     <script>
-      // See https://github.com/axemclion/grunt-saucelabs#test-result-details-with-qunit
-      var log = []
+      // Disable jQuery event aliases to ensure we don't accidentally use any of them
+      [
+        'blur',
+        'focus',
+        'focusin',
+        'focusout',
+        'resize',
+        'scroll',
+        'click',
+        'dblclick',
+        'mousedown',
+        'mouseup',
+        'mousemove',
+        'mouseover',
+        'mouseout',
+        'mouseenter',
+        'mouseleave',
+        'change',
+        'select',
+        'submit',
+        'keydown',
+        'keypress',
+        'keyup',
+        'contextmenu'
+      ].forEach(function(eventAlias) {
+        $.fn[eventAlias] = function() {
+          throw new Error('Using the ".' + eventAlias + '()" method is not allowed, so that Bootstrap can be compatible with custom jQuery builds which exclude the "event aliases" module that defines said method. See https://github.com/twbs/bootstrap/blob/master/CONTRIBUTING.md#js')
+        }
+      })
+
       // Require assert.expect in each test
       QUnit.config.requireExpects = true
-      QUnit.done(function (testResults) {
-        var tests = []
+
+      // See https://github.com/axemclion/grunt-saucelabs#test-result-details-with-qunit
+      var log = [];
+
+      QUnit.done(function(testResults) {
+        testResults.tests = [];
+
         for (var i = 0, len = log.length; i < len; i++) {
```

Comment:
Oh, if you look carefully, I haven't changed that line. Caching the length of an array used to speed the loop up a little, but it appears [Chrome does this optimisation automatically now](https://stackoverflow.com/questions/5752906/is-reading-the-length-property-of-an-array-really-that-expensive-an-operation), so we can actually remove it. I've changes to the test loader off into #22047.